main/ertp/api/amount-math.md

```diff
@@ -4,29 +4,27 @@ Logic for manipulating `amounts`.
 
 ## Obtaining an AmountMath
 
-There are three ways and circumstances you can get access to an `amountMath`:
-- You made its associated `mint` and `issuer` with `makeIssuerKit()`, so use the `amountMath` returned from the call. 
-- You are writing a Zoe contract and the `issuer` is saved in Zoe, so call `zcf.getAmountMath(brand)`. 
-- You receive or learn about an `issuer` and are not writing a Zoe contract, so call `makeLocalAmountMath(issuer)`.
+To get access to an `amountMath`:
+- `import { amountMath } from '@agoric/ertp';`
+  - It's a record, not an object with a constructor.
+- By writing a Zoe contract. The `issuer` is saved in Zoe, so call `zcf.getAmountMath(brand)`. 
 
 ## AmountMath Kinds
 
-There are three different kinds of `amountMath`, each of which implements all the methods shown on this page. You only have to specify the `amountMath` kind when creating its associated `issuer`.
+There are two different kinds of `amountMath`, each of which implements all the methods shown on this page. You only have to specify the `amountMath` kind when creating its associated `issuer`.
 
-The three kinds of `amountMath` each implement all of the same set of API methods (i.e. `amountMath` methods are polymorphic). We recommend you import the `MathKind` values from `@agoric/ERTP` instead of making the strings yourself. 
+The two`amountMath` kinds each implement all of the same API methods (i.e. `amountMath` methods are polymorphic). We recommend you import the `MathKind` values from `@agoric/ERTP` instead of making the strings yourself. 
```

Comment:
```suggestion
The two `amountMath` kinds each implement all of the same API methods (i.e. `amountMath` methods are polymorphic). We recommend you import the `MathKind` values from `@agoric/ERTP` instead of making the strings yourself. 
```